source/UberRides/Model/RideRequestLocation.swift

```diff
@@ -44,7 +44,7 @@
     public required init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         bearing = try container.decodeIfPresent(Int.self, forKey: .bearing) ?? 0
-        eta = try container.decodeIfPresent(Int.self, forKey: .eta) ?? 0
+        eta = try container.decodeIfPresent(Int.self, forKey: .eta) ?? -1
```

Comment:
Seems a bit weird to me that we initialize everything else as 0, but use a different "default" value here?

Reply:
Addressed in 9368f74. I converted them into Swift optionals, which won't be exposed to Objective-C. I then made additional getters and rename them for Obj-C so the end result is the same.

Instead of using a constant like `NEGATIVE_INFINITY`, I use -1, however, to remain consistent with our REST API. Thoughts?

Reply:
67845f7 uses objective-c constants instead of magic numbers, now.

Reply:
Yup consistency with API a good reason to do this.